ctapipe/io/astropy_helpers.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import tables
+from astropy.table import Table
+from astropy.units import Unit
+
+__all__ = ["h5_table_to_astropy"]
+
+
+def h5_table_to_astropy(h5file, path) -> Table:
+    """Get a table from a ctapipe-format HDF5 table as an `astropy.table.Table`
+    object, retaining units. This uses the same unit storage convention as
+    defined by the `HDF5TableWriter`, namely that the units are in attributes
+    named by `<column name>_UNIT` that are parsible by `astropy.units`.
+
+    Parameters
+    ----------
+    h5file: Union[str, tables.file.File]
+       input filename or PyTables file handle
+    path: str
+       path to table in the file
+
+    """
+
+    should_close_file = False
+    if not isinstance(h5file, tables.file.File) and isinstance(h5file, str):
+        h5file = tables.open_file(h5file)
+        should_close_file = True
+    else:
+        raise ValueError(
+            f"expected a string or PyTables filehandle for argument 'h5file', got {h5file}"
+        )
+
+    table: tables.table.Table = h5file.get_node(path)
```

Comment:
I really don't understand these kinds of type hints. What are they for? 

type hints only make sense in API definitions, right?

Reply:
it's not really needed, just allows my editor to do tab-completion without running the code. In principle it would also raise a mypy warning.  I will remove it, since not critical

Reply:
Ah, helps the editor, right.